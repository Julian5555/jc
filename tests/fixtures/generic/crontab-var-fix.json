{"variables":[],"schedule":[{"minute":["*"],"hour":["*"],"day_of_month":["*"],"month":["*"],"day_of_week":["*"],"command":"/bin/bash -l  -c \"python3 my_script.py --output-file=/opt/output.json\""}]}
