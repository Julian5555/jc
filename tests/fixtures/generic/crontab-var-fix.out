* * * * * /bin/bash -l  -c "python3 my_script.py --output-file=/opt/output.json"
